packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md

```diff
@@ -0,0 +1,155 @@
+5. In the section `IF Grant type is`, click the `Interaction Code` checkbox.
+
+### Enable `interaction code` grant on an application
+
+1. Navigate to `Applications > Applications` in the Admin UI
+2. Click an application in the list
+3. Click the `General` tab on the application detail
+4. Click edit on the `General Settings` panel
+5. In the section `Allowed grant types`, click the `Interaction Code` checkbox.
+
+## Web clients
+
+### Okta-hosted signin page (default)
+
+For most authentication flows that involve redirecting to Okta, there should be no other changes needed. Once the feature is enabled, the Okta Identity Engine will be used automatically, by default.
+
+### Customized signin page / custom domain
+
+For most users of the [custom domain](/docs/guides/custom-url-domain/overview/) feature, there are no other changes needed. The default template will detect and use the Okta Identity Engine automatically.
+
+However if you have [modified the template](https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/) in certain ways, such as to perform redirects or set cookies, these modificiations may not be compatible with the Okta Identity Engine. In particular, use of these methods and objects may not work with OIE:
+
+- setCookieAndRedirect
+- sessionToken
+
+In the default template, the `success` callback from `renderEl` is being handled by `OktaUtil.completeLogin`. This method correctly handles the authentication flow and we recommend using it.
+
+For reference, here is the default template:
+
+```html
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="none" />
+
+    <title>{{pageTitle}}</title>
+    {{{SignInWidgetResources}}}
+</head>
+<body>
+    <div class="login-bg-image" style="background-image: {{bgImageUrl}}"></div>
+    <div id="okta-login-container"></div>
+
+    <!--
+        "OktaUtil" defines a global OktaUtil object
+        that contains methods used to complete the Okta login flow.
+     -->
+    {{{OktaUtil}}}
+
+    <script type="text/javascript">
+        // "config" object contains default widget configuration
+        // with any custom overrides defined in your admin settings.
+        var config = OktaUtil.getSignInWidgetConfig();
+
+        // Render the Okta Sign-In Widget
+        var oktaSignIn = new OktaSignIn(config);
+        oktaSignIn.renderEl({ el: '#okta-login-container' },
+            OktaUtil.completeLogin,
+            function(error) {
+                // Logs errors that occur when configuring the widget.
+                // Remove or replace this with your own custom error handler.
+                console.log(error.message, error);
+            }
+        );
+    </script>
+</body>
+</html>
+```
+
+### Embedded signin widget
```

Comment:
Do we have a way to make this more clear that we mean the widget not on okta?  I know the first two categories cover the okta-hosted cases, but I had to scroll up and down a few times to be confident, and I assume man outside devs will be less familiar than I am.

Reply:
Maybe we could say something like "Apps which use the widget via node module or script tag" ?